README.md

```diff
@@ -1,10 +1,28 @@
 A Time Tracker 
 ==============
-Open Source Time Tracker for Android
+*Open Source Time Tracker for Android*
 
 
 - Based on TimeTracker+ by Sean E. Russell.
 - Application icon from Tango Project (tango.freedesktop.org)
 - Russian translation by Denis Gladkiy
 - French translation by Wilfried Kopp
 - Spanish and Italian translation by Carlos Manzo
+
+### Source code on [github](http://github.com/netmackan/ATimeTracker)
+
+### Install on android using [fdroid](http://f-droid.org/repository/browse/?fdfilter=atimetracker&fdid=com.markuspage.android.atimetracker)
+
+### Read the [docs](https://github.com/netmackan/ATimeTracker/blob/master/android-timetracker/docs/timetracker.rst)
+
+### Screenshots
+
+![help_screen](https://github.com/linuxcaffe/timew-android/blob/master/screenshots/help_screen.png) ![settings_screen](https://github.com/linuxcaffe/timew-android/blob/master/screenshots/settings_screen.png)
```

Comment:
I think it would make sense to use relative URLs for the images this way we would not have issues with the different forked repositories, i.e. netmackan vs linuxcaffe in the path [1].

[1] https://help.github.com/articles/relative-links-in-readmes/